platform/qt/src/qmapboxgl.cpp

```diff
@@ -703,7 +703,7 @@ double QMapboxGL::scale() const
 
 void QMapboxGL::setScale(double scale_, const QPointF &center)
 {
-    d_ptr->mapObj->setZoom(mbgl::util::log2(scale_), mbgl::ScreenCoordinate { center.x(), center.y() });
+    d_ptr->mapObj->setZoom(std::log2(scale_), mbgl::ScreenCoordinate { center.x(), center.y() });
```

Comment:
nit: s/`std::log2`/`::log2`/

Removing `std` namespace prevents us from compilation errors in older/buggy GCC compilers used in Qt CI.